### ISSUE TITLE
Docs: document reasons for avoiding GIF usage

### ISSUE THREAD

**Issue by jdbaldry** [grafana/writers-toolkit#182]
https://raintank-corp.slack.com/archives/C5PG2JK8W/p1674428027088579

**Comment by MattDodsonEnglish** [comment]
Here's the related WCAG guideline:

https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html
